core/templates/dev/head/domain/utilities/LanguageUtilService.js

```diff
@@ -60,6 +76,9 @@ oppia.factory('LanguageUtilService', [
     var audioLanguagesCount = allAudioLanguageCodes.length;
 
     return {
+      getLanguageIdsAndTexts: function() {
```

Comment:
Can we add a test for this in the existing test file?

Reply:
Done.